src/main/java/racingcar/domain/dao/MemoryGameHistoryDao.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.domain.dao;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+
+import javax.sql.DataSource;
+
+import racingcar.domain.GameHistory;
+import racingcar.repository.GameHistoryRepository;
+
+public class MemoryGameHistoryDao implements GameHistoryRepository {
+	private static final String TABLE_NAME = "game_history";
+	private static final String ID_COLUMN = "id";
+	private static final String DATE_COLUMN = "created_at";
+
+	private final Map<Long, GameHistory> store;
+	private final AtomicLong id;
+
+	public MemoryGameHistoryDao(DataSource dataSource) {
+		this.store = new ConcurrentHashMap<>();
+		this.id = new AtomicLong(1);
+	}
+
+	@Override
+	public void saveAll(List<GameHistory> gameHistories) {
+		gameHistories.stream()
+			.forEach(gameHistory -> store.put(gameHistory.getId(), gameHistory));
```

Comment:
증가하는 식별자를 부여하시려면 AtomicLong의 getAndIncreament() 메서드를 사용해주시면 됩니다 😄 

```suggestion
			.forEach(gameHistory -> store.put(id.getAndIncrement(), gameHistory));
```

Reply:
해당 클래스를 삭제하였습니다!